src/Cli/dotnet/commands/dotnet-sdk/check/LocalizableStrings.resx

```diff
@@ -145,7 +145,7 @@
     <value>Try out the newest .NET SDK features with .NET {0}.</value>
   </data>
   <data name="CommandFooter" xml:space="preserve">
-    <value>The latest versions of .NET can be installed from https://aka.ms/dotnet-core-download. For more information about .NET lifecycles, see https://aka.ms/dotnet-core-support.</value>
```

Comment:
Maybe it would be nice if source build maintainers could modify this entire string but I'm not sure how the config file approach would work with localization.  The reason I bring this up touches on a larger issue of how this feature works with package managers. RH and other linux users who install from package feeds, don't go to a url to install the product, they use the package manager to upgrade.  Does RH point their users to the package manager documentation?  That would be confusing in this case IMO.  Should they create a .NET upgrade page?  The best UX IMO would be if the message directed the user to the package manager.  

@omajid - thoughts?

Reply:
I agree.

For most Linux distributions, the job of `dotnet sdk check` is already done by the package manager  (`apt`/`apk`/`dnf`/`emerge`/`pacman`/`yum`/`zypper`. Unless we want to override that default workflow, `dotnet sdk check` ought to be a no-opt other than telling the users to use the distro-specific package manager.

Reply:
@MichaelSimons we could allow you to replace the entire string but I'm not sure we would be able to localize it then, is that a requirement for red hat? Alternatively I can add a switch to just not include this note, if you think most users would already know that they can update via package managers.

Reply:
I wonder if it's possible to override what `dotnet sdk check` does (in a Linux distro-package context). In particular, I would like if it didn't use the network at all - so distro maintainers wouldn't have to maintain a separate releases-index.json - and simply pointed users to run the package manager command to check/update.

Localization is nice-to-have, but making it work better out of the box for Linux distributions is probably more important.

If this note is not displayed, what does the `dotnet sdk check` command end up doing from a user's point of view?

Reply:
> I wonder if it's possible to override what dotnet sdk check does (in a Linux distro-package context). In particular, I would like if it didn't use the network at all - so distro maintainers wouldn't have to maintain a separate releases-index.json - and simply pointed users to run the package manager command to check/update.

If the ask is to only print out a message that points users to their package manager, we could do that by adding a config option for disabling the default output entirely, but anything beyond that would require source changes from the red hat team. And we would also have the same localization problem for any message we output from the config file. 

> If this note is not displayed, what does the dotnet sdk check command end up doing from a user's point of view?

This note is just a footer to the main bulk of the output. The output structure consists of a list of the installed runtimes and SDKs along with how up to date they are, followed by this message.

Reply:
@omajid can you please clarify the ask here? Right now the output essentially has 2 parts: the list of SDKs and runtimes followed by a message about how to update. Are you saying you would prefer not to have the first part at all and just have a message pointing users to their package managers?

Reply:
Thanks for explaining this.

In my ideal world, if a non-Microsoft-build of the SDK was installed using a package manager, showing the user results about the Microsoft SDK and the available versions doesn't make too much sense.

I think it would be better if, assuming the SDK was built and installed via a package manager, that the check command simply tell the user how to use the package manager to check/update. The package manager is the authoritative source of information in this scenario and any other data file (or URL) is either going to be incorrect or duplicating the canonical information in the package manager.

Hopefully my explanation makes sense.

My ask, assuming it's possible, is that `sdk check` work sensibly in such scenarios. (Not sure how to identify those, maybe via a build configuration?) Some options I can think of for what `sdk check` should do, in decreasing order of preference:

- `sdk check` simply tells users to use their package manager
  - maybe it even tells which commands to run and how?

OR

- `sdk check` uses the package manager internally and forwards what the package manager says

OR

- `sdk check` uses the `ReleasesUri` as in the original PR to find the Linux distro's set of versions and use that

What do you think, @MichaelSimons ?

Reply:
> sdk check simply tells users to use their package manager

If we assume that it's fine not to localize this message, then this can be done easily by modifying the config file in this PR to include a `CommandOutputReplacementString` that, when present, would replace the entire command output with that string.

Reply:
That works for me!

Reply:
Sounds good, any objections @MichaelSimons? Is there any reason to include the other items in the config file or should we switch to exclusively including this new `CommandOutputReplacementString`?

Reply:
I have no objections.  I still see value in having the url in the config file.

Reply:
Sounds good, I'll update this PR to have the following in the config file: 

```
{
    "ReleasesUri":null,
    "ReleasesFilePath":null,
    "CommandOutputReplacementString":"foo"
}
```

such that each option is mutually exclusive, only one should ever be populated.

Reply:
Just pushed this change, updating the PR description now, let me know if it looks okay